demos/config.h

```diff
@@ -22,9 +22,37 @@
 #ifndef CONFIG_H
 #define CONFIG_H
 
+
 /* Set network context to socket (int). */
 typedef int MQTTNetworkContext_t;
 
 #define MQTT_MAX_QUEUED_PUBLISH_MESSAGES 10
 
+/*
+ * @brief The Logging Interface Hook.
+ * 
+ * @note Customers can EITHER
+ * Define their port of the logging stack through this macro
+ *          OR
+ * Use the reference POSIX implementation of the logging stack, 
+ * IotLog_Generic (used by default)
+ * 
+ * @param[in] messageLevel The integer logging level code for the message.
+ * @param[in] pFormat The format string for the log message.
+ * @param[in] ... The variadic argument list for the format string.
+ **/
+#ifndef IotLog
```

Comment:
No need of ```#ifndef``` here as this is a project specific config file.

Reply:
Removed